apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-select.tsx

```diff
@@ -0,0 +1,40 @@
+import { IFormField, IValueAndDisplay } from '@coscrad/api-interfaces';
+import { MenuItem, Select } from '@mui/material';
+import { useState } from 'react';
+
+const NO_SELECTION_PLACEHOLDER = '-';
+
+interface VocabularyListSelectProps {
+    formField: IFormField;
+    onNewSelection?: (name: string, value: string | boolean) => void;
+}
+
+export const VocabularyListSelect = ({
```

Comment:
We may want to use the select instead of checkbox for booleans in order to have an indeterminate value.